md/licenses.md

```diff
@@ -52,11 +52,15 @@ The license file will be sent to you by email.
 When you receive the license file (`.lic` file extension):
 - If this is the first time you start the bundle, copy the file to the`<TOMCAT_HOME>/setup/platform_conf/licenses` folder or `<WILDFLY_HOME>/setup/platform_conf/licenses/` folder before starting the bundle.
 - If the bundle has already been started, [you need to use the Platform setup tool](BonitaBPM_platform_setup.md#update_platform_conf) to push your license file to database.
+Don't forget to restart your application server after pushing the license to the database.
 
 ## License renewal scheduling
 
 If you are still within the Subscription period when you approach the license expiration date, request a new license that starts on the last day of your current license.  
-Put the license you receive in the license folder alongside the existing license.  
+Put the license you receive in the license folder alongside the existing license. 
```

Comment:
This step should be preceded by this other one (to prevent ugly situations):
[Use the Platform setup tool](BonitaBPM_platform_setup.md#update_platform_conf) to pull the current configuration from the database.